README.md

```diff
@@ -1,2 +1,78 @@
-# jboss-eap-openshift-templates
-ImageStreams and Templates for the OpenShift Images of JBoss EAP
+# Red Hat JBoss Enterprise Application Platform XP 1.0 OpenShift container image imagestreams and templates
+
+## Importing Imagestreams and Templates
+
+You should have the 'oc' tools available and be logged in to your OpenShift instance. For more details on how to do this, please consult the OpenShift documentation.
+For example, for OpenShift Online, see: https://docs.openshift.com/online/cli_reference/get_started_cli.html
+[source, bash]
+----
+oc replace --force -f https://raw.githubusercontent.com/jboss-container-images/jboss-eap-openshift-templates/eap-xp1/jboss-eap-xp1-openjdk11-openshift.json
+
+for resource in eap-xp1-amq-s2i.json \
+  eap-xp1-basic-s2i.json \
+  eap-xp1-third-party-db-s2i.json
+do
+  oc replace --force -f https://raw.githubusercontent.com/jboss-container-images/jboss-eap-openshift-templates/eap-xp1/templates/${resource}
+done
+----
+
+If you have administrative access to the general `openshift` namespace and want the image streams and templates to be accessible by all projects, add -n openshift to the oc replace line of the command. For example:
+
+[source, bash]
+----
+oc replace -n openshift --force -f ...
+----
+
+#### Updating Existing Images
+To update an to the most recent image:
+
+[source, bash]
+----
+oc import-image jboss-eap-xp1-openjdk11-openshift:1.0
+oc import-image jboss-eap-xp1-openjdk11-runtime-openshift:1.0
+----
+
+Optionally include namespace to the import:
+[source, bash]
+----
+oc -n myproject import-image jboss-eap-xp1-openjdk11-openshift:1.0
+oc -n myproject import-image jboss-eap-xp1-openjdk11-runtime-openshift:1.0
+----
+
+#### Creating New Applications With Templates
+Example:
+
+[source, bash]
+----
+oc -n myproject new-app eap-xp1-basic-s2i -p APPLICATION_NAME=eap-xp1-basic-app
```

Comment:
is it correct? I thought you needed `--template=eap-xp1-basic-s2i` for templates?

according to the [https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.3/html/getting_started_with_jboss_eap_for_openshift_container_platform/build_run_java_app_s2i#import_imagestreams_templates](EAP 7.3 docs), I was expecting:

```
oc -n myproject new-app eap-xp1-basic-app --template=eap-xp1-basic-s2i
```